enterprise/server/invocation_stat_service/invocation_stat_service.go

```diff
@@ -546,8 +555,8 @@ type HeatmapQueryInputs = struct {
 	MetricArrayStr         string
 }
 
-func (i *InvocationStatService) generateQueryInputs(ctx context.Context, table string, metric string, q *stpb.TrendQuery, whereClauseStr string, whereClauseArgs []interface{}, timezoneOffsetMinutes int32) (*HeatmapQueryInputs, error) {
-	timestampBuckets, timestampArrayStr, err := getTimestampBuckets(q, time.Duration(timezoneOffsetMinutes)*time.Minute)
+func (i *InvocationStatService) generateQueryInputs(ctx context.Context, table string, metric string, q *stpb.TrendQuery, whereClauseStr string, whereClauseArgs []interface{}, requestContext *ctxpb.RequestContext) (*HeatmapQueryInputs, error) {
```

Comment:
not really related to this CL but this is a gnarly mf

Reply:
Sigh, yeah.  It's mostly structured this way for reuse, but I will take another pass at it someday.  It's not good when you already are thinking "who wrote this" a month after you write something.